### PR TITLE
Update Portal release notes

### PR DESCRIPTION
**Pull request by willllewellyn**

### CODE REVIEW

File: articles/portal/ptl-ref-release-notes.md [hunk 1]
```diff
@@ -19,6 +19,28 @@ toc_mdlink: ptl-ref-release-notes.md
...
 
+## Release 1.127.0
+
+Date: 18th February 2020
+
+Feature update
+
+Details:
+
+- When requesting a password reset, users that are locked will be send an email telling them that
```
Comment (SteveHallUKCLoud) [inline articles/portal/ptl-ref-release-notes.md]: Sent - not send

File: articles/portal/ptl-ref-release-notes.md [hunk 2]
```diff
@@ -377,21 +399,21 @@ Date: 17th May 2017
...
+Details: The 'List All Articles' button in the Knowledge Centre has now been moved to the top of the categories menu for better visibility.
 
 ## Release 1.55.0
 
 Date: 2nd May 2017
 
 Feature updates
 
-Details: Internet Bandwidth consumption showback available to all customers. Please see the Bandwidth Monitoring FAQ document in the Knowledge Centre for more details. 
+Details: Internet Bandwidth consumption showback available to all customers. Please see the Bandwidth Monitoring FAQ document in the Knowledge Centre for more details.
```
Comment (SteveHallUKCLoud) [inline articles/portal/ptl-ref-release-notes.md]: link to doc?

Comment (willllewellyn) [reply articles/portal/ptl-ref-release-notes.md]: That doc is three years old, I'm not convinced I'll be able to find it

Comment (willllewellyn) [reply articles/portal/ptl-ref-release-notes.md]: I think [that's](https://docs.ukcloud.com/articles/connectivity/conn-faq-bandwidth-monitoring.html?q=Bandwidth%20Monitoring%20FAQ) the closest thing we now have

File: articles/portal/ptl-ref-release-notes.md (hunk 1 above)
Comment (willllewellyn) [reply articles/portal/ptl-ref-release-notes.md]: Done here c4eb1b3

File: articles/portal/ptl-ref-release-notes.md (hunk 2 above)
Comment (suehighmoor) [reply articles/portal/ptl-ref-release-notes.md]: This is such an old item in the release notes, I don't think we need to bother providing a link

**Pull request merged**